### PR TITLE
Fix responsive for sidebar

### PR DESCRIPTION
**Pull request by tlnhan**

### CODE REVIEW

**Comment by tlnhan** [comment]
Hello, i checked. Hope its work. If its not work, tell me please.

File: packages/app-lixi/src/containers/Sidebar/SideBarShortcut.tsx [hunk 1]
```diff
@@ -327,10 +352,11 @@ const SidebarShortcut = () => {
                 onClick={() => router.push('/notifications')}
               />
             </Badge>
-            <div style={{ cursor: 'pointer' }} onClick={() => router.push(`/profile/${selectedAccount?.address}`)}>
+            <div style={{ cursor: 'pointer' } && { marginBottom: '2vh' }} onClick={() => router.push(`/profile/${selectedAccount?.address}`)}>
               <AvatarUser name={selectedAccount?.name} isMarginRight={false} />
             </div>
           </UserControl>
+>>>>>>> Stashed changes
```
Comment (vince8x) [inline packages/app-lixi/src/containers/Sidebar/SideBarShortcut.tsx:359]: Please remove merge message

File: packages/app-lixi/src/containers/Sidebar/SideBarShortcut.tsx [hunk 2]
```diff
@@ -313,6 +336,8 @@ const SidebarShortcut = () => {
               </>
             )} */}
           </div>
+<<<<<<< Updated upstream
```
Comment (vince8x) [inline packages/app-lixi/src/containers/Sidebar/SideBarShortcut.tsx:339]: Please remove merged message

File: packages/app-lixi/src/containers/Sidebar/SideBarShortcut.tsx [hunk 3]
```diff
@@ -67,9 +67,15 @@ export const ContainerAccess = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  height: 100vh;
+  height: 100%;
   background: linear-gradient(0deg, rgba(158, 42, 156, 0.08), rgba(158, 42, 156, 0.08)), #fffbff;
+<<<<<<< Updated upstream
```
Comment (vince8x) [inline packages/app-lixi/src/containers/Sidebar/SideBarShortcut.tsx:72]: Please remove incorrect merge message

File: packages/app-lixi/src/containers/Sidebar/SideBarShortcut.tsx [hunk 4]
```diff
@@ -67,9 +67,15 @@ export const ContainerAccess = styled.div`
...
   border-right: 1px solid #f4e3f4;
+  overflow: -moz-scrollbars-none;
+  -ms-overflow-style: none;
+  .item-access {
+=======
```
Comment (vince8x) [inline packages/app-lixi/src/containers/Sidebar/SideBarShortcut.tsx:77]: Please remove incorrect merge message

**Comment by khanhvinhnguyen** [comment]
Please check @media rule when do we use nav bar and when do we use sidebar
Screenshot 2023-03-13 at 1 52 03 AM

**Pull request closed without merging**